static/bidder-info/zentotem.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "https://rtb.zentotem.net/bid?sspuid=cqlnvfk00bhs0b6rci6g"
```

Comment:
![image](https://github.com/user-attachments/assets/7b81822e-0eb4-4f67-9c3d-fe1a3109e381)
it doesnt work